### ISSUE TITLE
crashes when providing an existing path, because this is == and not !=

### ISSUE THREAD

**Issue by Bendimester23** [prisma/prisma-client-go#767]
https://github.com/prisma/prisma-client-go/blob/7cd44d572f0fdc1e43c9a071ee1f9ee253dd76e7/engine/lifecycle.go#L98

**Comment by steebchen** [comment]
Oh.. thanks! I added a fix via #767

Can you please check if it works?

```
go get github.com/prisma/prisma-client-go@steebchen-patch-2
# re-generate client
# re-try
```

**Comment by Bendimester23** [comment]
![image](https://user-images.githubusercontent.com/58264102/188306172-3a778cf6-8e41-483a-a78e-6ac44f09ab5e.png)
This issue is fixed, however it still can't start.

**Comment by Bendimester23** [comment]
After looking at the code, I think I found the proble. https://github.com/prisma/prisma-client-go/blob/f022062736fdd652f06336082e07bcaaf8838eb5/engine/lifecycle.go#L114 This check runs every time and it overwrites the query engine path.

**Comment by steebchen** [comment]
Thanks for looking into this, this wasn't tested at all after the initial version and I broke it probably quite early along the way.. Lacking tests as well unfortunately

I pushed another update to fix what you said in the previous version, can you please try again?

**Comment by Bendimester23** [comment]
I does work.

**Comment by steebchen** [comment]
Nice. Thanks for testing :D

**Comment by steebchen** [comment]
released in v0.16.2
